lib/ShareByCircleProvider.php

```diff
@@ -450,12 +450,12 @@ private function editShareEntry($data) {
 			$name = $data['circle_alt_name'];
 		}
 
-		$data['share_with'] =
-			sprintf(
-				'%s (%s, %s) [%s]', $name,
-				$this->l10n->t(Circle::TypeLongString($data['circle_type'])),
-				$this->miscService->getDisplayName($data['circle_owner'], true), $data['share_with']
-			);
+//		$data['share_with'] =
```

Comment:
this isn't actually the display name, you'd need to put it into "share_with_displayname" instead, if useful